actions/hypervisor.find.empty.yaml

```diff
@@ -16,5 +16,10 @@ parameters:
     enum:
       - "dev"
       - "prod"
+  include_offline:
```

Comment:
why `include_offline` rather than `include_down_state_hvs` - I think its more informative

Reply:
Following our discussion in-person, do you still want me to rename this?

`down` doesn't really work here, as it's ambiguous if the HV marked down for scheduling or is uncontactable

In this case, it's referring to HVs that are not contactable, e.g. offline, nova not installed, or even decommissioned and physically removed

I'm not sure the best way to name this that way?

Reply:
I'm going to leave this as is - we can look at it later if its a problem